emoji-ui/src/components/Pagination.js

```diff
@@ -23,7 +23,10 @@ class PaginationBase extends Component {
 
   _onInputChange(e) {
     const { setCurrentPage } = this.props.paginationActions;
-    setCurrentPage(e.target.value);
+    const pageNum = parseInt(e.target.value);
+    if (isNaN(pageNum)) {
```

Comment:
sorry, i will change this to `setCurrentPage(isNaN(pageNum) ? 0 : pageNum);` so i can delete all the numbers. otherwise, there will always have 1 number that can not be deleted.